README.md

```diff
@@ -239,6 +240,35 @@ TestCafe developers and community members made these plugins:
 * **ESLint**<br/>
   Use ESLint when writing and editing TestCafe tests.
   * [ESLint plugin](https://github.com/miherlosev/eslint-plugin-testcafe) (by [@miherlosev](https://github.com/miherlosev))
+  
+## Different versions of TestCafe
+
+There is a line of products called `TestCafe`. Below are the similarities and key differences between them.
+
+* All three versions share the same core features:
+  * No need for WebDriver, browser plugins or other tools.
+  * Cross-platform and cross-browser out of the box.
+
+* [**TestCafe**](https://testcafe.devexpress.com/)<br/>
+  *2013, commercial web application*
```

Comment:
maybe add *first released in* `*first released in 2013, commercial web application*`